src/components/ProductPage.jsx

```diff
@@ -0,0 +1,51 @@
+import React, {useState} from 'react';
+import ProductTable from './ProductTable';
+
+const ProductPage = () => {
+   let [inStockOnly, setInStockOnly] = useState(false);
+   let [count, setCount] = useState(0);
+
+  const handleIncrement = () => {
+    console.log('hitting increment');
+    count++;
+    setCount(count);
```

Comment:
You can combine these two lines like so:
```suggestion
    setCount(count++);
```